contributing.md

```diff
@@ -0,0 +1,83 @@
+# Contributing
+
+GO-SHIP welcomes codes and contributions from anyone who feels they can benefit the
+community.
+
+This document is designed to guide those who have a new code that they wish to bring
+to the GO-SHIP Organisation.
+For details on how to contribute to an existing repository please see the documentation
+of that specific repository for guidance.
+
+
+## Pre-requisites
+
+If you have a code that you would like to add to the organisation please
```

Comment:
Should we specify "if you have a code in any programming language (e.g. Python, R, MatLab, etc.)" ?

Reply:
Done.